include/tvm/tir/schedule/state.h

```diff
@@ -142,6 +142,8 @@ class ScheduleStateNode : public Object {
   /******** Property of blocks ********/
   /*! \brief Returns the BlockInfo correpsonding to the block sref */
   TVM_DLL BlockInfo GetBlockInfo(const StmtSRef& block_sref) const;
+  /*! \brief Recalculate the BlockInfo recursively under stmt */
+  TVM_DLL void UpdateSubtreeBlockInfo(const Stmt& stmt);
```

Comment:
Do we have any additional constraint of this API? For example:
- `stmt` must be a `Block`?
- All loop /blocks in the subtree must not have any sref

Reply:
The function just updates the BlockInfo. I think it is always doable to do so.
If we want to enforce some constraint, then the constraint should be all loop/blocks in the subtree must have sref.